# The Cubic Polynomials Path/Trajectory Data Type.
# This is used for defining a trajectory using cubic polynomials equations

# path type
string path_type

# inital location
float64 qi_x
float64 qi_y
float64 qi_theta

# final location
float64 qf_x
float64 qf_y
float64 qf_theta

# total travel time
float64 T

# cubic polynomial parameter
float64 poly_k
